Misc/NEWS.d/next/Library/2022-03-10-14-51-11.bpo-46968.ym2QxL.rst

```diff
@@ -0,0 +1 @@
+Dynamically determine size of sigaltstack CPython allocates during initialization of the faulthandler extension to ensure Linux signal delivery on the hardware running the interpreter. The required Linux kernel support is available since kernel version 5.14.
```

Comment:
I would prefer to put faulthandler at the beginning. Like:

```
:mod:`faulthandler`: On Linux 5.14 and newer, dynamically determine size (...) using ``getauxval(AT_MINSIGSTKSZ)`` (...).
```

Rather than "sigaltstack", I suggest to say: "signal handler stack size".

IMO it's worth it to go into the details:

"This changes allows for Python extension's request to Linux kernel
to use AMX_TILE instruction set on Sapphire Rapids Xeon processor
to succeed, unblocking use of the ISA in frameworks."

nitpick: Can you please try to limit to 80 columns?

Reply:
I modified the .rst file directly as requested.